lib/gestures.dart

```diff
@@ -13,6 +13,8 @@ mixin MultiTouchTapDetector on Game {
 class DragEvent extends Drag {
   Offset initialPosition;
```

Comment:
this can be final right?

Reply:
You are right. The analyzer didn't see that as it is not private.